### ISSUE TITLE
Adding a toggle button for switching between light and dark theme

### ISSUE THREAD

**Issue by SoumyadipGhosh23** [Anupkjha2601/finalproj#59]
Hey!! I would like to add a dark theme to your website to enhance user experience and accommodate user preferences.

By implementing a light and dark theme toggle button, users will have the flexibility to personalize their browsing experience according to their preference for either a brighter or darker interface. The light theme option ensures a clean and vibrant layout, perfect for daytime usage or well-lit environments. On the other hand, the dark theme option creates a visually appealing and comfortable browsing experience, especially for users in low-light conditions or for those who prefer a more subdued aesthetic.

**Comment by Saurabhrajput1234** [comment]
yes, hi sir can solve this issue to develop a dark theme toggle button.(GSSoc23).
thanks you
